db/blip_handler.go

```diff
@@ -942,6 +942,9 @@ func (bh *blipHandler) handleProveAttachment(rq *blip.Message) error {
 		if bh.clientType == BLIPClientTypeSGR2 {
 			return ErrAttachmentNotFound
 		}
+		if IsMissingDDocError(err) {
```

Comment:
This is intended for checking missing design docs (ddoc) - you want to use IsKeyNotFound, I think.

Reply:
Sure, that makese sense, I didn't realize this earlier, thank you, Adam.